e2e/wdio/core/tests/flexible-column-layout.e2e-spec.ts

```diff
@@ -90,6 +92,14 @@ describe('Flexible column layout component test', () => {
     it('should check separator for dynamic example', () => {
         checkSeparator(dynamicExample);
     });
+    // skipped due to https://github.com/SAP/fundamental-ngx/issues/6518
+    xit('should check missed separator for default example', () => {
```

Comment:
Are we going to add tests that are initially skipped? In that case, we need to open a second pr to enable them. Maybe we should wait till bugs will be fixed and then add enabled tests?

Reply:
@platon-rov yes, we add them now and they can either be unskipped by the PR that fixes the issue OR in a separate PR if it's missed.